projects/distributed-tracing/src/shared/dashboard/interaction/detail-sheet/detail-sheet-interaction-handler.service.ts

```diff
@@ -5,7 +5,9 @@ import {
   DETAIL_SHEET_INTERACTION_MODEL
 } from './container/detail-sheet-interaction-container.component';
 
-@Injectable()
+@Injectable({
+  providedIn: 'root'
```

Comment:
likewise, shouldn't be in root if it's providing its own injector

Reply:
I have to understand this. You can explain above

Reply:
took it back above.